src/core/blockservice.ts

```diff
@@ -62,9 +63,9 @@ export class BlockService {
    */
   async get(cid: CID): Promise<Block> {
     if (this.exchange != null) {
-      return this.exchange.get(cid)
+      return this.exchange.get(new Old(Buffer.from(cid.buffer)));
```

Comment:
This is because we still have to interact with code that is expecting the old 'cids' module. Should probably catch this in the new multiformats library (I'll create a ticket there).